content/conditions/hernia/surgery-guide/section-2.md

```diff
@@ -1,3 +1,7 @@
+### Approx 6 weeks{.label}
```

Comment:
As you say, the divider is just visual, I don't think it's semantically a `hr`. Maybe it should be re-ordered to `h2` then `h3` though but we might have to move the border to the top h2 to show the visual separation.